<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChillerWindowDisplay</class>
 <widget class="QWidget" name="ChillerWindowDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Чиллер ОМБ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	
	
	background-color: rgb(101, 99, 95);
	
}

QPushButton
{
background-color: rgb(82, 82, 82);
color: rgb(213, 199, 165);
	font: 75 11pt &quot;Arial&quot; bold;
}

QPushButton:checked 
{ 
background-color: green; 
}

QGroupBox
{
border: 0px solid line black;
	background-color: rgb(136, 131, 122,10);
}
QLabel
{
border: 1px solid line black;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 233, 202);
}
QSpinBox
{
font: 12pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 233, 202);
}

QSlider
{
border: 1px solid line black;
	background-color: rgb(205, 166, 153);
}

QListWidget
{
border: 1px solid line black;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 233, 202);
}

QTableWidget
{
border: 1px solid line black;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 233, 202);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel
{
border: 1px solid line black;
	background-color: rgb(85, 170, 127);
}</string>
       </property>
       <property name="text">
        <string>Датчики температуры</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>7</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="butOpenPort">
          <property name="text">
           <string>Подключить</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="boxPortList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listChillerOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="ButSendCommand">
       <property name="font">
        <font>
         <family>Arial 13</family>
         <pointsize>11</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Пуск</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame
{
border: 1px solid line black;
	
background-color: rgb(101, 99, 95);;
}

QLabel
{
border: 1px solid line black;
	background-color: rgb(82, 82, 82);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinSetTemperatureLevel">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>25</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
border: 0px solid line black;
	background-color: rgba(148, 74, 74, 0);
	color: rgb(220, 204, 189);
}</string>
          </property>
          <property name="text">
           <string>Температура</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="butReset">
       <property name="text">
        <string>Перезапуск</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel
{
border: 1px solid line black;
	background-color: rgb(205, 171, 143);
}</string>
     </property>
     <property name="text">
      <string>Чиллер ОМБ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
